playground/vite.config.ts

```diff
@@ -1,14 +1,17 @@
 import { defineConfig } from 'vite'
 import Vue from '@vitejs/plugin-vue'
 import Inspect from 'vite-plugin-inspect'
+import { ElementPlusResolver } from 'unplugin-vue-components/resolvers'
 import AutoImport from '../src/vite'
+import { componentResolver } from '../src'
 
 export default defineConfig({
   plugins: [
     Vue(),
     Inspect(),
     AutoImport({
       imports: ['vue', '@vueuse/core'],
+      resolvers: [componentResolver(ElementPlusResolver())],
```

Comment:
Could we possibly make it just work without the converting function?

Reply:
This is a bit difficult, because components and directives need to be handled separately.

Reply:
@antfu Done.